data-raw/qc-markdowns/adult-holding-redd-and-carcass-surveys/feather-river/feather-carcass-survey-qc.Rmd

```diff
@@ -0,0 +1,539 @@
+#   rename(sec = "sect")
+```
+
+### Counts
+
+The `feather_carcass_chops_and_tags_combined` table contains carcass counts by chop/tagged based on clips
+
+## Explore Numeric Variables: {.tabset}
+
+### Variable: `id`, `chan_id`, `sect`, `min`
+
+```{r}
+feather_carcass_chops_and_tags_combined %>% 
+  select_if(is.numeric) %>%
+  colnames()
+```
+
+```{r}
+summary(feather_carcass_chops_and_tags_combined$id)
+```
+```{r}
+summary(feather_carcass_chops_and_tags_combined$chan_id)
+```
+```{r}
+summary(feather_carcass_chops_and_tags_combined$sec)
+```
+
+```{r}
+summary(feather_carcass_chops_and_tags_combined$min)
+```
+
+### Chop Channel Variable: `chop_count`, `tag_count`
+
+
+```{r}
+summary(feather_carcass_chops_and_tags_combined$chop_count)
+```
+
+```{r}
+summary(feather_carcass_chops_and_tags_combined$tag_count)
+```
+
+**NA and Unknown Values**
+Provide a stat on NA or unknown values.
+
+```{r}
+round(sum(is.na(feather_carcass_chops_and_tags_combined$chop_count))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100
+round(sum(is.na(feather_carcass_chops_and_tags_combined$tag_count))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100
+round(sum(is.na(feather_carcass_chops_and_tags_combined$id))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100
+round(sum(is.na(feather_carcass_chops_and_tags_combined$chop_clip_status))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100
+round(sum(is.na(feather_carcass_chops_and_tags_combined$tag_clip_status))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100
+round(sum(is.na(feather_carcass_chops_and_tags_combined$count))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100
+```
+* `r round(sum(is.na(feather_carcass_chops_and_tags_combined$chop_count))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100` % of values in the `chop_count` column are NA.
+* `r round(sum(is.na(feather_carcass_chops_and_tags_combined$tag_count))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100` % of values in the `tag_count` column are NA.
+* `r round(sum(is.na(feather_carcass_chops_and_tags_combined$id))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100` % of values in the `id` column are NA.
+* `r round(sum(is.na(feather_carcass_chops_and_tags_combined$chop_clip_status))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100` % of values in the `chop_clip_status` column are NA.
+* `r round(sum(is.na(feather_carcass_chops_and_tags_combined$tag_clip_status))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100` % of values in the `tag_clip_status` column are NA.
+* `r round(sum(is.na(feather_carcass_chops_and_tags_combined$count))/nrow(feather_carcass_chops_and_tags_combined), 3) * 100` % of values in the `count` column are NA.
+
+
+**Plotting chops over Period of Record**
+
+```{r}
+feather_carcass_chops_and_tags_combined %>% 
+  mutate(date = as_date(date)) %>% 
+  ggplot(aes(x = date, y = chop_count)) +
+  geom_col() +
```

Comment:
maybe keep on same scale so easier to see difference. though if it looks weird then nevermind. could summarize differences in words.